conf/application.conf

```diff
@@ -229,7 +229,7 @@ direct-debit-frontend {
 
 vat-sign-up-frontend {
   host = "http://localhost:9566"
-  signUpUrl = "/vat-through-software/sign-up/your-vat-number"
+  reSignUpUrl = "/vat-through-software/sign-up/vat-number/"
```

Comment:
Could you also remove the other things that reference the now-removed `signUpUrl` such as in `ConfigKeys` and `frontendAppConfig`.